AirCasting/SDSync/FixedSessions/SDCardFixedSessionsSavingService.swift

```diff
@@ -75,19 +75,43 @@ class SDCardFixedSessionsUploadingService {
         }
     }
     
-    private func enqueueForUploading(measurements: SDCardMeasurementsRow, buffer streamsWithMeasurements: inout [SDStream: [Measurement]]) {
+    private func provideLines(of url: URL, progress: (FileLineReaderProgress) -> Void) throws {
+        var isDirectory: ObjCBool = false
+        guard FileManager.default.fileExists(atPath: url.path, isDirectory: &isDirectory) else { progress(.endOfFile); return }
+        
+        if !isDirectory.boolValue {
+            try self.fileLineReader.readLines(of: url, progress: progress)
+        } else {
+            let files = try FileManager.default.contentsOfDirectory(atPath: url.path).compactMap({ url.path + "/" + $0 }).compactMap(URL.init(string:))
+            Log.info("Files: \(files)")
```

Comment:
I'd change this log to make it more distinct and easy to read from the log file later. Something like `Reading all file lines from directory at ${path}. Files count: ${files_count}`. We are listing the files already in the next log, so no need to duplicate that.